infra/variables.tf

```diff
@@ -74,3 +70,26 @@ variable "registries" {
     default = ["mock-receiver"]
 }
 
+variable "vpc_id" {
+    default = "vpc-0f91c7ddfc1af661a"
+}
+
+variable "nlb_ports" {
```

Comment:
Are these variables needed in infra? looks like these are ECS specific. If they're not used in infra they don't belong here